src/api/models/__init__.py

```diff
@@ -8,6 +8,7 @@
 from .base import Country  # noqa: F401
 from .base import JSONMixin  # noqa: F401
 from .base import Profile  # noqa: F401
+from .base import validate_max_year  # noqa: F401
```

Comment:
I think if we're moving field validation out of models, we should make that clear by removing field validators and moving functions like this into utils or resources.